source/code/shared/tools/scx_ssl_config/scx_ssl_config.cpp

```diff
@@ -471,8 +471,12 @@ static int DoGenerate(const wstring & targetPath, int startDays, int endDays,
                 throw SCXCoreLib::SCXErrnoFileException(L"chown", keyPath.Get(), errno, SCXSRCLOCATION);
             }
         } 
-        else if (errno != 0) {
-           throw SCXCoreLib::SCXErrnoException(L"getpwnam", errno, SCXSRCLOCATION);
+        else if(errno !=0 && errno !=ENOENT && errno !=ESRCH) {
+            const char* isTestEnv = getenv("SCX_TESTRUN_ACTIVE");
+            if(isTestEnv)
```

Comment:
Suggest just throwing an exception if `! isTestEnv`. Realize that it's perfectly normal on build systems (and DEV systems) to not have an OMI account.

If you want to do more, I'd suggest that you just log the error at 477 (via logging, which I think is enabled here) rather than use wcout, which will clutter the unit test output.